src/service/mercury/helpers/transformers.ts

```diff
@@ -116,9 +116,13 @@ const transformAccountBalances = async (
   }, {} as NonNullable<AccountBalancesInterface["balances"]>);
 
   const classicBalances = classicBalanceData.reduce((prev, curr) => {
+    const codeAscii = Buffer.from(
```

Comment:
unrelated but wanted to sneak this in here as it steps around some Mercury parsing for asset codes.